img_diff.py

```diff
@@ -43,7 +43,7 @@ def compare(image1, image2, threshold):
     return [p2 if diff(p1, p2) else (0, 0, 0) for p1, p2 in zip(pixels1, pixels2)]
 
 
-def get_size(image1, image2):
+def get_size(image1: str, image2: str) -> int:

```

Comment:
Image.size actually returns a tuple containing two ints.

Reply:
Fixed

Reply:
Fixed in the second commit